### DISCUSSION TITLE
Config file

### DISCUSSION THREAD

**Discussion by wreczek** [astral-sh/ruff#3798] [Q&A] [answered]
Hello, I can't find the configuration file. I wanted to change the line length value from 88 to 100. Google doesn't say anything about it, nor does the documentation. I want the changes to be global, not just for a specific project.

**Comment by charliermarsh** [comment, marked as answer]
Hey! You can create a user-specific configuration file based on the instructions here: https://beta.ruff.rs/docs/faq/#how-can-i-change-ruffs-default-configuration.

E.g., put the following at `/Users/Alice/Library/Application Support/ruff/ruff.toml`:

```toml
line-length = 100
```
